### DISCUSSION TITLE
Is it safe to add purely cosmetic elements outside the body?

### DISCUSSION THREAD

**Discussion by Steeeves** [tinymce/tinymce#8440] [General]
The app I'm working on displays the Editor content on a TV and currently I'm trying to match the style of the Editor to the style on the TV.

This means I have divisions and other elements that are completely independent of the Editor's content on the TV and I want to replicate them inside the Editor.

- - -

Although it is not a good practice, I noticed if I add elements inside the Editor's between the `html` and the `body` element, I'm able to "style" the Editor's document without actually affecting the content inside. For example, I can recreate the header and footer that envelops the content on the TV in the Editor.

Additionally, I also noticed that there is a `export_ignore_elements` option, however, from what I could tell it completely ignores all the elements by name, which is not what I want. Maybe if there was a way to ignore elements by classes I could make use of it.

- - -

I guess my question here is, is it safe to add elements outside the Editor's body? If not what other options do I have to style the Editor without affecting the content?
From what I've tested it looks fine, but there is no guarantee that it won't break in the future.

**Comment by TheSpyder** [comment]
It's not something we really support, but all that means is if you find a problem they need to be turned off before confirming it's a TinyMCE bug.

We do it ourselves for the table resize bars:

![Screenshot 2023-02-13 at 11 15 34 am](https://user-images.githubusercontent.com/298292/218345942-f5dfe218-510d-4ac1-9e24-da48af29951d.png)

**Reply by Steeeves** [reply]
Alright, thanks for the feedback!
